jekyll/_cci2/sample-config.md

```diff
@@ -225,3 +225,70 @@ workflows:
             - precompile_assets
 ```
 {% endraw %}
+
+## Sample iOS App Configuration with Code Signing and Crashlytics
+In the following example configuration the test steps run with the specified commands.
```

Comment:
Need to use the example from [here](https://circleci.com/docs/2.0/ios-migrating-from-1-2/) with all the options discussed above (swiftenv, etc) added